openedx/features/discounts/applicability.py

```diff
@@ -82,6 +82,30 @@ def get_discount_expiration_date(user, course):
     return discount_expiration_date
 
 
+def AA759_can_show_coupon(user, course):
```

Comment:
if we decide to implement the experiment this could probably be made more general - but we'd need to do more investigation with product and revenue team around the different business rules for different coupons

Reply:
At minimum, lowercase the AA to fit normal python expectations.

But I'd also personally call this something like `can_show_streak_discount` or something. And then its docstring does the explaining of the ticket name and such.

Reply:
how about can_show_streak_experiment_coupon?  
i'm fine removing the ticket in this case, since in this case there is a docstring immediately next to it